packages/wallets/src/components/Base/WalletDropdown/WalletDropdown.scss

```diff
@@ -35,19 +35,32 @@
     }
 
     &__field {
+        position: absolute;
+        inset: 0;
         min-width: 0; /* this is required to reset input's default width */
+        padding-left: 4rem;
         display: flex;
         flex-grow: 1;
         font-family: inherit;
         outline: 0;
         font-size: 1.4rem;
+        background-color: transparent;
         color: var(--system-light-2-general-text, #333);
         transition: border-color 0.2s;
-        cursor: pointer;
-    }
+        cursor: unset;
+        user-select: none;
+
+        &::selection {
+            background-color: transparent;
+        }
 
-    &__field::placeholder {
-        color: transparent;
+        &::-moz-selection {
+            background-color: transparent;
+        }
```

Comment:
```suggestion
```

Reply:
Yea, we don't really have those for mozilla in `wallets` anywhere

<img width="370" alt="Screenshot 2023-11-15 at 02 49 18" src="https://github.com/binary-com/deriv-app/assets/119863957/3ea98285-6486-445e-bae2-0da223138d15">